table_refresh.go

```diff
@@ -15,25 +15,22 @@ import (
 // This limit exists because we can only generate 'maxCplForRefresh' bit prefixes for now.
 const maxCplForRefresh uint = 15
 
-// CplRefresh contains a CPL(common prefix length) with the host & the last time
-// we refreshed that cpl/searched for an ID which has that cpl with the host.
-type CplRefresh struct {
-	Cpl           uint
-	LastRefreshAt time.Time
-}
-
 // GetTrackedCplsForRefresh returns the Cpl's we are tracking for refresh.
 // Caller is free to modify the returned slice as it is a defensive copy.
-func (rt *RoutingTable) GetTrackedCplsForRefresh() []CplRefresh {
+func (rt *RoutingTable) GetTrackedCplsForRefresh() []time.Time {
```

Comment:
This function signature change is fine and only sends back the information we actually need. In the future we may want to change the information returned to the DHT when it wants to decide if/when to refresh a bucket, but since go-libp2p-kad-dht is the only real user of this I guess we can just change the interface back later 🤷‍♂.

Reply:
It just felt a bit strange to return the bucket numbers as well.

Reply:
ya, I get it that's why 🤷‍♂